test/testVizkit3dWorld.cpp

```diff
@@ -13,17 +12,14 @@ BOOST_AUTO_TEST_CASE(it_loads_correctly_a_sdf_world)
     BOOST_CHECK_EQUAL(robot_viz.begin()->first,"simple_model");
 }
 
-BOOST_AUTO_TEST_CASE(it_creates_two_widgets)
+BOOST_AUTO_TEST_CASE(it_can_create_multiple_vizkit3_world)
 {
-    vizkit3d_world::Vizkit3dWorld *vizkit3dWorld;
-    vizkit3dWorld = new vizkit3d_world::Vizkit3dWorld(
-        "test_data/simple.world",
-        std::vector<std::string>(),
-        std::vector<std::string>(),
-        800, 600,
-        60.0,
-        0.01, 1000.0,
-        2);
-    BOOST_TEST(vizkit3dWorld->getWidget(0)->isWindow(),1);
-    BOOST_TEST(vizkit3dWorld->getWidget(1)->isWindow(),1);
+    vizkit3d_world::Vizkit3dWorld vizkit3d_world("test_data/simple.world");
```

Comment:
Initialize the app before:
```suggestion
    QApplication* app;
    int argc = 1;
    char const*argv[] = { (char *)"vizkit3d_world" };
    app = new QApplication(argc, const_cast<char**>(argv));
    vizkit3d_world::Vizkit3dWorld vizkit3d_world("test_data/simple.world");
```